projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/ProjectCode.java

```diff
@@ -72,7 +74,7 @@ public static ProjectCode parse(String str) throws IllegalArgumentException {
   }
 
   private static boolean isValid(String code) {
-    return code.startsWith(PREFIX) && (code.length() == LENGTH);
+    return code.startsWith(PREFIX) && (code.length() == LENGTH) && !containsInvalidCharacters(code);
```

Comment:
Why do we need to check for invalid characters here as well? In line 66 there is this check already done :)

Reply:
Good point, this method was used differently before - now it is unused and parse(str) takes over all validation. It would save some space, but I think the name is wrong if it does not check for all statements that make a code illegal. Would you agree that we can remove the method?

Reply:
the isValid method?